modules/storage-api/src/main/java/org/apache/ignite/internal/storage/engine/MvTableStorage.java

```diff
@@ -78,6 +79,16 @@ public interface MvTableStorage {
     @Nullable
     SortedIndexStorage getSortedIndex(int partitionId, String indexName);
 
+    /**
+     * Returns an already created Hash Index with the given name or {@code null} if it does not exist.
+     *
+     * @param partitionId Partition ID for which this index has been built.
+     * @param indexName Index name.
+     * @return Hash Index storage or {@code null} if it does not exist..
+     */
+    @Nullable
+    HashIndexStorage getHashIndex(int partitionId, String indexName);
```

Comment:
```suggestion
    /**
     * Returns an already created Hash Index with the given name or {@code null} if it does not exist.
     *
     * @param partitionId Partition ID for which this index has been built.
     * @param indexName Index name.
     * @return Hash Index storage or {@code null} if it does not exist.
     */
    @Nullable HashIndexStorage getHashIndex(int partitionId, String indexName);
```

Reply:
I personally don't like this formatting, sorry